extensions/wikia/ExactTargetUpdates/api/ExactTargetApi.php

```diff
@@ -113,13 +116,45 @@ public function makeDeleteRequest( Array $aCallObjects, $sObjectType = 'DataExte
 	 */
 	protected function sendRequest( $sType, $oRequestObject ) {
 		try {
-			$oResults = $this->client->$sType( $oRequestObject );
-			WikiaLogger::instance()->info( $this->client->__getLastResponse() );
+			$oResults = $this->getClient()->$sType( $oRequestObject );
+			$this->getLogger()->info( $this->getClient()->__getLastResponse() );
 			return $oResults;
-		} catch ( SoapFault $e ) {
-			/* Log error */
-			WikiaLogger::instance()->error( __METHOD__ . "::{$sType} SoapFault: " . $e->getMessage() . ' ErrorCode:  ' . $e->getCode() );
+		} catch ( \SoapFault $e ) {
+			$this->getLogger()->error( __METHOD__ . "::{$sType} SoapFault: " . $e->getMessage() . ' ErrorCode:  ' . $e->getCode() );
```

Comment:
Let's log only the `__METHOD__` as the message and move the exception to the context as `['exception' => $e]`.